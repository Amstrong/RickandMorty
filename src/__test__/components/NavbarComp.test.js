import React from "react";
import { Link } from "react-router-dom";
import renderer from "react-test-renderer";
import { StaticRouter } from "react-router";

describe("<Navbar/>", () => {
  test("Verifying the ReactRouter Link", () => {
    const component = renderer.create(
      <StaticRouter location="someLocation" context={{}}>
        <Link to="/" />
      </StaticRouter>
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
