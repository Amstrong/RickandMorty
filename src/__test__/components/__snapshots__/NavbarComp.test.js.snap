// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar/> Verifying the ReactRouter Link 1`] = `
<a
  href="/"
  onClick={[Function]}
/>
`;
